docker/conf/docker-compose.yml.dist

```diff
@@ -94,6 +94,14 @@ services:
         environment:
             - discovery.type=single-node
 
+    kibana:
```

Comment:
Shouldn't be docker/conf/docker-compose-mac.yml.dist updated too?

Reply:
Wow, thanks a lot for noticing!